givitsite/templates/feed.html

```diff
@@ -0,0 +1,30 @@
+{%extends 'base.html'%}
+
+{%block content%}
+
+<h1>Feed</h1>
+<table border='1'>
+<tr>
+<td>Title</td>
+<td>City</td>
+<td>URL</td>
+<td>Phone Number</td>
+<td>Pic</td>
+<td>Match</td>
+</tr>
+{% for item in founds%}
+
+<tr>
+<td>{{item.title}}</td>
+<td>{{item.city}}</td>
+<td><a href="{{item.url}}">link</a></td>
+<td>{{item.phone_number}}</td>
+<td><img src="{{item.picture}}" width='50px'/></td>
+<td><input type='button' value='match'/></td>
+</tr>
+
+{%endfor%}
+</table>
+
+
+{%endblock%}
```

Comment:
"no newline at end of file" here too. :mask:

Reply:
fixed